### DISCUSSION TITLE
Qiita記事が技術力スコアに反映されるタイミングについて

### DISCUSSION THREAD

**Discussion by K-yokosu** [lapras-inc/lapras-user-community#20] [Q&A] [answered]
スコアに反映されているかどうかがわかりません、、、

技術力スコアにはいいね数が反映されるとされていますが、Teck Articleのスコアが微動だにしません。
やったこと
・記事数3→9
・いいね数 2→31
・Teck Article 2.31→2.31

この程度ではスコアは変動しないよということであれば頑張るだけなんですが、その辺の匙加減が分からないので皆さんの状況をお聞きしたいです。

**Comment by UserSuccessTeamLAPRAS** [comment, marked as answer]
いつもLAPRASをご利用いただきありがとうございます。

先んじて、スコアの説明に誤りがありましたことをお詫び申し上げます。

現在、スコアの説明には「ZennとQiitaへのいいね数をスコアとする」と記載されておりますが、Qiitaの場合は「いいね数」ではなく「ストック数」をスコアとしております。
また、すべての記事が対象となるわけではなく、ストック数が多い上位3つの記事のストック数の合計がスコアに反映されます。

記事数についてもご懸念いただいているかと存じますが、スコアには記事数は反映されない仕様となっております。

なお、表示されるスコアは、アウトプット活動をLAPRAS内のランキングに基づいて相対的に計算しているものです。そのため、ユーザー様がアウトプット活動をされたとしても、相対的にスコアが変動しない場合もございます。

今後ともLAPRASをよろしくお願いします。

LAPRASユーザーサポート

**Comment by rockymanobi** [comment]
「いいね」の考慮のためのトピック立ちました
https://github.com/lapras-inc/lapras-user-community/discussions/22
